main.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'app'
+
+def display_list
+  action_list = {
+    '1': 'List all books',
+    '2': 'List all people',
+    '3': 'Create a person',
+    '4': 'Create a book',
+    '5': 'Create a rental',
+    '6': 'List all rentals for a given person id',
```

Comment:
- [ ] As it is a [requirement ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_ui.md)for this project, "`List all rentals for a given person id.`", please ensure one is able to list the rentals for a particular `id`, that is not the case at the moment, please kindly fix that. What I get is an empty list.